Lint-Whoppers/nutshell
# Cj news delete button

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions

git fetch --all
git checkout cjNewsDeleteButton

Although I made this branch to get my delete button working...for now...just check to make sure that News Articles show on the DOM as soon as the page loads without you having to click anything.



# Checklist:
